### ISSUE TITLE
Add Reminder to Configure Layer 2 After BAP/BPP Setup in Beckn ONIX

### ISSUE THREAD

**Issue by emmayank** [beckn/beckn-onix#210]
## Description
By default, the BAP client does not have Layer 2 configuration schemas, which results in a 422 error when users try to make requests without configuring Layer 2. After successfully setting up BAP/BPP in Beckn ONIX, a message should be displayed to remind users to "Add Layer2 config before using the BAP/BPP" to prevent such errors.

## Goals
- [x] Ensure users are reminded to configure Layer 2 after BAP/BPP setup to avoid potential errors.
- [x] Improve the user experience by providing clear post-setup instructions.

## Expected Outcome
1. After successful setup of BAP/BPP, a message is displayed reminding users to add the Layer 2 config before using the BAP/BPP.
2. The message should appear immediately after the success message to ensure visibility.

## Acceptance Criteria
- [x]  After setting up BAP, a message is displayed: "Add Layer2 config before using the BAP."
- [x]  After setting up BPP, a message is displayed: "Add Layer2 config before using the BPP."
- [x]  The message appears directly after the success message for both BAP and BPP setups.
- [x]  The code changes are reviewed and approved by the development team.
- [x]  The installation process is tested to ensure the message is displayed correctly without affecting functionality.

## Mockups / Wireframes
N/A

## Product Name
Beckn ONIX 

## Domain
N/A

## Tech Skills Needed
Devops

## Complexity
High

## Category
Deployment

## Sub Category
Installation

**Comment by prasad-takale-eminds** [comment]
I made the necessary changes to the code. Below is the screenshot of the BAP installation:

![Image](https://github.com/user-attachments/assets/1558cbc1-fe42-445e-b092-0a34aebb66b6)

After the changes, I connected with @vbabuEM and reviewed it. As per his suggestion, we need to prompt the user for layer configuration during the BAP/BPP installation. He has created a new issue for this: #222.
